index.html

```diff
@@ -1,6 +1,15 @@
+<!DOCTYPE html>
 <html>
+
 <head>
-<script defer src="script.js"></script>
+    <title>Scratch Assets Image Viewer</title>
+    <script defer src="script.js"></script>
 </head>
-<body></body>
-</html>
+
+<body>
+    <h1>Scratch Assets Image Viewer</h1>
+    <p>To view an image, simply paste the URL into the box below.</p>
+    <input type="url" onchange="loadURL(this)">
```

Comment:
Adding an event-listener in the JavaScript is better.

Reply:
but then I'd have to give it an id.